docs/spelling_wordlist.txt

```diff
@@ -5,3 +5,9 @@ scm
 Threadless
 threadless
 youtube
+socio
+sexualized
+https
+www
+html
```

Comment:
This will probably be unnecessary if you fix the document to have an explicit link by wrapping that link with angled brackets.